yaml2sbml/YamlModel.py

```diff
@@ -80,8 +80,19 @@ def write_to_yaml(self,
 
         reduced_model_dict = self._get_reduced_model_dict()
 
+        #translate to string
+        yaml_as_string = yaml.dump(reduced_model_dict,
+                                   sort_keys=False,
+                                   indent=6)
```

Comment:
Please correct me if i'm wrong, but i'm assuming that `indent` is the number of spaces used for each indentation, in this case it would be 6. Isn't the default 4 spaces?

Reply:
As far as I remember, the indentation is currently 2 spaces, which looks awful. I used 6 in order for it to match the examples and the format definition. :)